src/components/projects/ManuscriptPageContainer.tsx

```diff
@@ -71,9 +75,22 @@ const ManuscriptPageContainer: React.FC = () => {
 
 const ManuscriptPageView: React.FC = () => {
   const can = usePermissions()
-
   const [trackingVisible, toggleTrackingVisibility] = useTrackingVisibility()
-  const isTrackingVisible = !can.editWithoutTracking && trackingVisible
+  const [{ view }] = useStore((store) => ({ view: store.view }))
+
+  // Get tracking status from editor state
+  const trackChangesStatus = view?.state
+    ? trackChangesPluginKey.getState(view.state)?.status
+    : TrackChangesStatus.disabled
+
+  const showTrackChangesToggle =
+    !can.editWithoutTracking &&
+    trackChangesStatus !== TrackChangesStatus.viewSnapshots
+
+  const isTrackingVisible =
```

Comment:
It could be `const isTrackingVisible = showTrackChangesToggle && trackingVisible` to avoid repetition.